src/components/MyAccount/Settings/AccountSettingsTab.tsx

```diff
@@ -34,56 +34,54 @@ const AccountSettingsTab = ({ settingsData }: { settingsData: Patron }) => {
 
   const submitAccountSettings = async (e) => {
     e.preventDefault()
+    setIsLoading(true)
     const payload = parsePayload(e.target, mostRecentPatronData)
-    try {
-      const response = await fetch(
-        `${BASE_URL}/api/account/settings/${mostRecentPatronData.id}`,
-        {
-          method: "PUT",
-          headers: {
-            "Content-Type": "application/json",
-          },
-          body: JSON.stringify(payload),
-        }
-      )
-      const responseData = await response.json()
-      if (response.ok) {
-        setMostRecentPatronData(updatePatronData(mostRecentPatronData, payload))
-        setCurrentlyEditing(false)
-        setModalProps(successModalProps)
-        openModal()
-      } else {
-        alert(`error: ${responseData}`)
-        setModalProps(failureModalProps)
-        openModal()
+    const response = await fetch(
+      `/research/research-catalog/api/account/settings/${mostRecentPatronData.id}`,
+      {
+        method: "PUT",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(payload),
       }
-    } catch (error) {
-      alert("fetching error")
+    )
+    if (response.status === 200) {
+      setMostRecentPatronData((prevData) => updatePatronData(prevData, payload))
+      setCurrentlyEditing(false)
+      setModalProps(successModalProps)
+      openModal()
+    } else {
+      setModalProps(failureModalProps)
+      openModal()
     }
+    setIsLoading(false)
   }
 
-  return (
-    <Box className={styles.accountSettingsTab}>
+  return isLoading ? (
+    <SkeletonLoader showImage={false} />
+  ) : (
+    <>
       {modalProps && <Modal {...modalProps} />}
       <Form
+        className={styles.accountSettingsTab}
         id="account-settings-container"
         onSubmit={(e) => submitAccountSettings(e)}
       >
-        <AccountSettingsButtons
-          currentlyEditing={currentlyEditing}
-          setCurrentlyEditing={setCurrentlyEditing}
-        />
         <List
           sx={{ border: "none", h2: { border: "none" } }}
           className={styles.myAccountList}
           type="dl"
         >
           {listElements}
         </List>
-        <Spacer />
+        <Spacer display={{ base: "none", md: "inline-block" }} />
+        <AccountSettingsButtons
```

Comment:
Button should be aligned with the top of the form

Reply:
fixed